dev/com.ibm.ws.webcontainer.security.admin/src/com/ibm/ws/webcontainer/security/admin/internal/WebAdminSecurityConfigImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2011, 2019 IBM Corporation and others.
+ * Copyright (c) 2011, 2019, 2020 IBM Corporation and others.
```

Comment:
Only specify two dates, the first and last ... `2011, 2020`

Reply:
done